src/test/kotlin/ui/graphicmusicnotation/details/MusicScoreDetailsContentUiTest.kt

```diff
@@ -0,0 +1,157 @@
+            runComposeUiTest {
+                val testComponentContext: ComponentContext = getTestComponentContext()
+                val graphicProperties =
+                    GraphicProperties(
+                        offsetX = 87.5,
+                        offsetY = 400.0,
+                        measureWidth = 500.0,
+                        spaceBetweenMeasures = 75.0,
+                        cutOffXToReflectNoteIsEnding = 0.0,
+                        wholeStepExpressedInY = 50.0
+                    )
+                val pathDataRepository = PathDataRepositoryImpl(graphicProperties)
+                val musicScoreDetailsComponent: MusicScoreDetailsComponent =
+                    DefaultMusicScoreDetailsComponent(testComponentContext, pathDataRepository)
+
+                setContent { MusicScoreDetailsContent(musicScoreDetailsComponent) }
+
+                val initialPathData = pathDataRepository.getPathData()
+                initialPathData.shouldBeEmpty()
+
+                onNodeWithTag("score-dropdown-menu-load").assertDoesNotExist()
+                onNodeWithTag("score-dropdown-menu").performClick()
+                onNodeWithTag("score-dropdown-menu-load").assertExists()
+                onNodeWithTag("score-dropdown-menu-load").performClick()
+                onNodeWithText("Load Frère Jacques").assertExists()
+                onNodeWithText("Load Frère Jacques").performClick()
+
+                val resultPathData = pathDataRepository.getPathData()
+
+                resultPathData shouldBe
+                    getExampleScoreFrereJacques()
+                        .parts[0]
+                        .measures
+                        .asPathData(pathDataRepository.getGraphicProperties())
+            }
+        }
+
+        "Loading an existing score while a score was already in progress should set new PathData" {
+            runComposeUiTest {
+                val testComponentContext: ComponentContext = getTestComponentContext()
+                val graphicProperties =
+                    GraphicProperties(
+                        offsetX = 87.5,
+                        offsetY = 400.0,
+                        measureWidth = 500.0,
+                        spaceBetweenMeasures = 75.0,
+                        cutOffXToReflectNoteIsEnding = 0.0,
+                        wholeStepExpressedInY = 50.0
+                    )
+                val pathDataRepository = PathDataRepositoryImpl(graphicProperties)
+                val pathData: List<PathNode> =
+                    PathBuilder().moveTo(x = 87.5f, y = 550.0f).horizontalLineTo(x = 587.5f).nodes
+                pathDataRepository.replacePathData(pathData)
+                val musicScoreDetailsComponent: MusicScoreDetailsComponent =
+                    DefaultMusicScoreDetailsComponent(testComponentContext, pathDataRepository)
+
+                setContent { MusicScoreDetailsContent(musicScoreDetailsComponent) }
```

Comment:
Probably this throws an exception on GHA runner because it can't open the MidiDevice.